docs-for

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```

Comment:
We have a kind of "standard prelude" we use for scripts in reflex-platform: https://github.com/reflex-frp/reflex-platform/blob/de5dee744ec37b648eaaeed7ee0a0d917a443e31/try-reflex#L1-L6

I think we should probably use that here as well.